src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1970,7 +1974,13 @@ def _maybe_expand_transformer_param_shape_or_error_(
         is_peft_loaded = getattr(transformer, "peft_config", None) is not None
         for name, module in transformer.named_modules():
             if isinstance(module, torch.nn.Linear):
-                module_weight = module.weight.data
+                is_quantized = module.weight.__class__.__name__ == "Params4bit"
```

Comment:
```suggestion
                is_bnb_4bit_quantized = module.weight.__class__.__name__ == "Params4bit"
```

I think better to be more explicit as we have a couple of quantization backends.